src/components/Tasks/TaskEdit.tsx

```diff
@@ -391,7 +362,7 @@ class TaskEdit extends React.PureComponent<PropsType> {
           onOk={() => this.props.onDelete(this.props.item!, this.props.initialCollection!)}
           onCancel={() => this.setState({ showDeleteDialog: false })}
         >
-        Are you sure you would like to delete this event?
+          Are you sure you would like to delete this event?
```

Comment:
Please, no unrelated styling changes. Additionally, please change "event" to "task".